tests/test_derivatives.py

```diff
@@ -332,15 +332,15 @@ def test_fd_new_side(self):
 
     @pytest.mark.parametrize('so, expected', [
         (2, 'u(x)/h_x - 1.0*u(x - 1.0*h_x)/h_x'),
-        (4, '1.125*u(x)/h_x + 0.0416666667*u(x - 2.0*h_x)/h_x - '
-            '1.125*u(x - 1.0*h_x)/h_x - 0.0416666667*u(x + 1.0*h_x)/h_x'),
-        (6, '1.171875*u(x)/h_x - 0.0046875*u(x - 3.0*h_x)/h_x + '
-            '0.0651041667*u(x - 2.0*h_x)/h_x - 1.171875*u(x - 1.0*h_x)/h_x - '
-            '0.0651041667*u(x + 1.0*h_x)/h_x + 0.0046875*u(x + 2.0*h_x)/h_x'),
-        (8, '1.19628906*u(x)/h_x + 0.000697544643*u(x - 4.0*h_x)/h_x - '
-            '0.0095703125*u(x - 3.0*h_x)/h_x + 0.0797526042*u(x - 2.0*h_x)/h_x - '
-            '1.19628906*u(x - 1.0*h_x)/h_x - 0.0797526042*u(x + 1.0*h_x)/h_x + '
-            '0.0095703125*u(x + 2.0*h_x)/h_x - 0.000697544643*u(x + 3.0*h_x)/h_x')])
+        (4, '1.125*u(x)/h_x + 0.0416666667*u(x - 2*h_x)/h_x - '
```

Comment:
yes, but I mean, this makes no difference because what you see in this test are partially evaluated derivatives... the "+- h_x" will be replaced with "+- 1" (or whatever offset) before code generation. It's an array index in the end, so it must be an integer...